embedded-graphics/src/mock_display.rs

```diff
@@ -349,6 +365,21 @@ where
 
         display
     }
+
+    /// TODO: Docs
+    pub fn from_points<I>(points: I) -> Self
+    where
+        I: Iterator<Item = Point>,
+    {
+        let mut display = MockDisplay::new();
+
+        points
+            .map(|p| Pixel(p, C::on()))
```

Comment:
I like the general idea of having `ColorMapping::on`, but I'm not sure if we need this here. My idea for `from_points` was only to use it for testing points iterators against patterns, which doesn't require other colors. And if you need that feature you could also use `map` to get any color: `MockDisplay::from_points(p).map(|_| Rgb888::MAGENTA)`.

Or we could implement `MockDisplay::draw_points(&mut self, points: I, color: C)` instead. This would allow us to combine multiple points iters into a single mock display pattern. This would also have the advantage that you could disable the overdraw and out of bounds checks before drawing.

Reply:
See other comment